cnf-tests/testsuites/pkg/namespaces/namespaces.go

```diff
@@ -115,6 +115,12 @@ func Create(namespace string, cs *testclient.ClientSet) error {
 	return err
 }
 
+// Delete delete a new namespace with the given name.
+func Delete(namespace string, cs *testclient.ClientSet) error {
```

Comment:
I don't think we need this.

we already have (https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/testsuites/e2esuite/test_suite_test.go#L157)

Reply:
done